packages/request-node/test/getConfirmedTransaction.test.ts

```diff
@@ -60,7 +65,7 @@ describe('getConfirmedTransaction', () => {
     expect(serverResponse!.body.result).toMatchObject({});
     // 'getConfirmedTransaction request meta'
     expect(serverResponse!.body.meta.storageMeta.state).toBe('confirmed');
-  }, 11000);
+  }, 30000);
```

Comment:
Are the 19 additional seconds needed just for the mining?

Reply:
indexing of the graph. I increased a bit more than actually needed, to avoid flakiness